### ISSUE TITLE
mvn clean install -Dmaven.test.skip=true  is error

### ISSUE THREAD

**Issue by sd4324530** [MartinWeindel/presto-kudu#8]
I compiled this source code, but it was wrong
command：mvn clean install -Dmaven.test.skip=true
Error log:
[ERROR] Failed to execute goal com.facebook.presto:presto-maven-plugin:0.3:generate-service-descriptor (default-generate-service-descriptor) on project presto-kudu: Execution default-generate-service-descriptor of goal com.facebook.presto:presto-maven-plugin:0.3:generate-service-descriptor failed: A required class was missing while executing com.facebook.presto:presto-maven-plugin:0.3:generate-service-descriptor: ml\littlebulb\presto\kudu\KuduClientConfig (wrong name: ml/littlebulb/presto/kudu/KuduClientConfig)
[ERROR] -----------------------------------------------------
[ERROR] realm =    plugin>com.facebook.presto:presto-maven-plugin:0.3
[ERROR] strategy = org.codehaus.plexus.classworlds.strategy.SelfFirstStrategy
[ERROR] urls[0] = file:/D:/repository/com/facebook/presto/presto-maven-plugin/0.3/presto-maven-plugin-0.3.jar
[ERROR] urls[1] = file:/D:/repository/com/google/guava/guava/14.0.1/guava-14.0.1.jar
[ERROR] urls[2] = file:/D:/repository/com/facebook/presto/presto-spi/0.76/presto-spi-0.76.jar
[ERROR] urls[3] = file:/D:/repository/io/airlift/slice/0.7/slice-0.7.jar
[ERROR] urls[4] = file:/D:/repository/com/fasterxml/jackson/core/jackson-annotations/2.4.2/jackson-annotations-2.4.2.jar
[ERROR] urls[5] = file:/D:/repository/com/fasterxml/jackson/core/jackson-databind/2.4.2/jackson-databind-2.4.2.jar
[ERROR] urls[6] = file:/D:/repository/com/fasterxml/jackson/core/jackson-core/2.4.2/jackson-core-2.4.2.jar
[ERROR] urls[7] = file:/D:/repository/org/codehaus/plexus/plexus-utils/3.0.17/plexus-utils-3.0.17.jar
[ERROR] Number of foreign imports: 1
[ERROR] import: Entry[import  from realm ClassRealm[project>ml.littlebulb.presto.kudu:presto-kudu:0.198.2-SNAPSHOT, parent: ClassRealm[maven.api, parent: null]]]
[ERROR] 
[ERROR] -----------------------------------------------------
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/PluginContainerException

**Comment by MartinWeindel** [comment]
I just downloaded the current sources, started with an empty ~/.m2 repository and built the project according to the build instructions on the `README.md`:
```bash
mvn -DskipTests clean package
```
Build was successful on my Linux machine. I'm using Apache Maven 3.5.0.
I have also tried your build command `mvn clean install -Dmaven.test.skip=true`, but this also worked without any problems.

So sorry, I have no idea what's your problem.
